### PR TITLE
Add GKE Resource Consumption Metering, promote resource export to GA

### PR DESCRIPTION
**Pull request by rileykarson**
Fixes https://github.com/terraform-providers/terraform-provider-google/issues/4634, supersedes https://github.com/GoogleCloudPlatform/magic-modules/pull/2705

Test changes are a little messy, I got rid of a conditional block and made it two cases instead.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
container: added `resource_usage_export_config` to `google_container_cluster`, previously only available in `google-beta` (ga only)
```

```release-note:enhancement
container: added `enable_resource_consumption_metering` to `resource_usage_export_config` in `google_container_cluster` (beta only)
```

### CODE REVIEW

**Comment by morgante** [comment]
I think this file also needs to be updated: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/validator/container.go

Somewhat off topic, but I thought we had some CI which would detect drift in manually written resources for conversion?

File: third_party/terraform/tests/resource_container_cluster_test.go.erb [hunk 1]
```diff
@@ -3257,21 +3264,31 @@ resource "google_container_cluster" "with_ip_allocation_policy" {
...
+}
+
+resource "google_container_cluster" "with_resource_usage_export_config" {
+  name               = "%s"
+  location           = "us-central1-a"
+  initial_node_count = 1
   resource_usage_export_config {
     enable_network_egress_metering = true
-
+		enable_resource_consumption_metering = %s
```
Comment (megan07) [inline third_party/terraform/tests/resource_container_cluster_test.go.erb]: nit: indentation

**Comment by rileykarson** [comment]
@morgante: Filed https://github.com/terraform-providers/terraform-provider-google/issues/5988 to resync container cluster, https://github.com/terraform-providers/terraform-provider-google/issues/5989 to warn us again

**Comment by morgante** [comment]
Thanks @rileykarson!

**Pull request merged**